components/pkg-export-kubernetes/src/manifestjson.rs

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) 2017 Chef Software Inc. and/or applicable contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use failure::SyncFailure;
+use handlebars::Handlebars;
+use serde_json::Value;
+
+use export_docker::Result;
+
+use manifest::Manifest;
+use error::Error;
+
+// Kubernetes manifest template
+const MANIFESTFILE: &'static str = include_str!("../defaults/KubernetesManifest.hbs");
+const BINDFILE: &'static str = include_str!("../defaults/KubernetesBind.hbs");
+
+pub struct ManifestJson {
+    pub main: Value,
+    pub binds: Vec<Value>,
```

Comment:
Do these need to be public?

Reply:
Yes, at least helm exporter needs to use them in chart.rs to split the manifest into a manifest template and values files.

Reply:
but I'm ok with adding simple getter methods if that's more appropriate.

Reply:
Public is fine.